src/main/java/org/myrobotlab/io/FileIO.java

```diff
@@ -870,33 +868,22 @@ public static void main(String[] args) throws ZipException, IOException {
    *          Python/examples/someFile.py
    * @return byte array
    */
-  @Deprecated /* user Service.getResource(src) */
   static public final byte[] resourceToByteArray(String src) {
 
-    // this path assumes in a jar ?
-    // String filename = "/resource/" + src;
-    log.info("looking for Resource {}", src);
+    log.info("looking for resource {}", src);
     InputStream isr = null;
-    if (isJar()) {
-      // this path assumes in a jar ? ensure it's forward slashes
-      String filename = "/resource/" + src.replace("\\", "/");
-      isr = FileIO.class.getResourceAsStream(filename);
-    } else {
-      String localFilename = Service.getResourceRoot() + File.separator + src;
-      try {
-        isr = new FileInputStream(localFilename);
-      } catch (Exception e) {
-        Logging.logError(e);
-        log.error("File not found. {}", localFilename, e);
-        return null;
-      }
+    String resource = ConfigUtils.getResourceRoot();
+    String localFilename = resource + File.separator + src;
+    try {
+      isr = new FileInputStream(localFilename);

```

Comment:
yeah, its not really "inside" the jar...
default action for mrl on startup is to extract the /resource from the jar and put it on the file system
This is the way war files worked many moons ago, so getting resources doesn't need to contend with the challenges of using a zip filesystem.   The new ConfigUtils.getResourceRoote() gets the info from RuntimeConfig.resource and nowhere else.  This gives us the flexibility to set that value, but default it will point to the extracted /resource